Engine/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ set(PYENGINE_HEADER PySide2_Engine_Python.h)
 set(POST_SHIBOKEN ../tools/utils/runPostShiboken2.sh)
 
 set(shiboken_args
-    "--enable-parent-ctor-heuristic" "--use-isnull-as-nb_nonzero"
+    "-std=c++17" "--enable-parent-ctor-heuristic" "--use-isnull-as-nb_nonzero"
```

Comment:
is it really "-std" with a single hyphen?

Reply:
Yes. If you run "shiboken2 --help" you'll see that I can either use -std= or --language-level=. The mix of single and double '-' named options is a little weird, but my guess is that this is just so it matches g++ and clang's option.